api/src/wfl/api/spec.clj

```diff
@@ -56,9 +57,11 @@
 
 ;; compound
 (s/def ::items (s/or :aou (s/+ ::items-aou)
-                     :wgs (s/+ ::items-wgs)))
+                     :wgs (s/+ ::items-wgs)
+                     :xx  ::items-xx))
 (s/def ::workflows (s/or :aou (s/* ::workflow-aou)
-                         :wgs (s/+ ::workflow-wgs)))
+                         :wgs (s/+ ::workflow-wgs)
+                         :xx  (s/+ ::workflow-xx)))
```

Comment:
Where is the schema for `workflow-xx` defined?

Reply:
good point!